packages/ui-library/src/common/index.ts

```diff
@@ -1,3 +1,35 @@
 export { createStyles } from '@mui/material/styles';
 export { useTheme, useMediaQuery } from '@mui/material';
 export * from './notistack';
+
+export const fallbackCopyTextToClipboard = (text: string) => {
```

Comment:
I'm not a fan of having a function in this file, I believe it should only be for exporting sibling files/directories. Maybe it's time to have a common/utils directorory (utils package coming soon 😄 )